bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/GJets_bj_LO_MLM/GJets_bbj_DR-0p4/GJets_bbj_DR-0p4_proc_card.dat

```diff
@@ -0,0 +1,37 @@
+#************************************************************
```

Comment:
one last thing: please remove the banners in the proc_card since this is not VERSION 1.5.11